tests/testthat/test-clean-names.R

```diff
@@ -184,7 +184,21 @@ test_that("locale-specific make_clean_names tests", {
 test_that("do not create duplicates (fix #251)", {
   expect_equal(
     make_clean_names(c("a", "a", "a_2")),
```

Comment:
Nit: this `a_2` was to test a collision, it should now be `a_1` to collide with the new result on the second `a`.  I believe the result on the next line would be `c("a", "a_2", "a_1")`

Reply:
Other two resolved, but this one I'm not sure. I still believe this should result in `c("a", "a_1", "a_2")` -- the only duplicates are the two versions of "a". Should we add an additional test for `c("a", "a", "a_1")` --> `c("a", "a_2", "a_1")` or change _this_ test?

Reply:
I went ahead and added another test to force this new type of collision. Kept the other one for protection of a "normal" no dupes case.

Reply:
Perfect.